docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(data.shape)
</pre><p>Is there a reason to print the data.shape? I would add a comment why you print the data.shape.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>